cooltools/lib/common.py

```diff
@@ -9,6 +9,70 @@
 import bioframe
 
 
+def assign_view_2D(
+    features,
+    view_df,
+    cols=["chrom1", "start1", "end1", "chrom2", "start2", "end2"],
+    cols_view=["chrom", "start", "end"],
+    features_view_cols=["region1", "region2"],
+    view_name_col="name",
+    drop_unassigned=False,
+):
+    """Assign region names from the view to each feature
```

Comment:
add clarification that this function assigns the "regular"(for lack of a better work) view to each both sets of intervals `cols1` and `cols2`
and it does not work with a "proper" 2D view - as that one would need to be defined using a pair of intervals